creativecloud/scripts/scripts.js

```diff
@@ -190,3 +190,5 @@ const miloLibs = setLibs(LIBS);
   loadLana({ clientId: 'cc' });
   await loadArea();
 }());
+
+export default CONFIG;
```

Comment:
Can we avoid exporting config from script and import getConfig from miloLibs where needed?

Reply:
yeah. we do not need it now. Removed it.